### PR TITLE
integrating PPHW last comment

### PR DESCRIPTION
**Pull request by bam241**

### CODE REVIEW

**Comment by bam241** [comment]
sorry I should maybe have done one commit per modification... (I'll try to do it next time!)

**Comment by bam241** [comment]
I will soon integrate the new plots (corresponding to the new calculation) and re-organize them! but the text is there.

**Comment by bam241** [comment]
@gonuke I have finish to include your remarks.

File: EG23.tex [hunk 1]
```diff
@@ -465,154 +440,117 @@ \subsection{Case 1: Greedy Separations}
...
 
-\begin{figure}[h!]
-    \centering
-    \subfigure[Used Fuel in Cooling Storage\label{fig:UFCS_1}]  {\epsfig{figure=img/usedFuelInCooling,width=0.48\textwidth}}
-    \subfigure[Fuel waiting for Reprocessing\label{fig:FWR_1}]  {\epsfig{figure=img/UNFWaitingReprocessing_1,width=0.48\textwidth}}
-    \caption{Used fuel in cooling and reprocessing.\label{fig:cool_reprocc} }
-\end{figure}
-
 The amounts of the different fuels in ``Used Fuel in Cooling Storage'' (Figure
 \ref{fig:UFCS_1}) follow the discharge of the different reactors as expected.
```
Comment (gonuke) [inline EG23.tex]: This figure no longer exists

File: EG23.tex [hunk 2]
```diff
@@ -465,154 +440,117 @@ \subsection{Case 1: Greedy Separations}
...
-\ref{fig:deploy_sub} and \ref{fig:power_sub} match perfectly with results from the
-DYMOND simulations. The time scale in the output spreadsheet filled starts in 2015
-and ends in 2215, while the DYMOND calculation used as reference for the
-deployment schedule ends in 2210. The sudden drop in 2210 is due to a lack of
-deployment data after 2210.
-
-%% NOTE: are we supposed to plot beyond 2210? or 2200?
-%% It looks like Bo only plotted to 2210 to avoid this drop at the end... is that hard to change?
+The generated power and the deployment schedule correspond as expected to the
+required one (shown on Figures \ref{fig:deploy} and \ref{fig:power} ) match
```
Comment (gonuke) [inline EG23.tex]: should be fig:nrg instead of fig:power

File: EG23.tex [hunk 3]
```diff
@@ -465,154 +440,117 @@ \subsection{Case 1: Greedy Separations}
...
+one can still observe a large plutonium accumulation spike 2100 and 2145 that
+could be avoided with a better reprocessing capacity deployment.
 
 \begin{figure}[h!]
     \centering
-    \subfigure[Annual Reprocessing Rate\label{fig:ARR_2}]   {\epsfig{figure=img/AnnualReprocessingRate_2,width=0.48\textwidth}}
-    \subfigure[Fuel waiting for reprocessing]               {\epsfig{figure=img/UNFWaitingReprocessing_2,width=0.48\textwidth}}
-    \subfigure[Fuel Resource in Storage\label{fig:SFC_2}]  {\epsfig{figure=img/FuelInStorage_2,width=0.48\textwidth}}
-    \caption{Resources production and fuel loading.\label{fig:ARR_FWR_SFC_2} }
+    \subfigure[Case 1.\label{fig:ARR_1}] {\epsfig{figure=img/AnnualReprocessingRate_1,width=0.48\textwidth}}
```
Comment (gonuke) [inline EG23.tex]: These plots occur twice... one of them should be "Fuel in Cooling Storage" or so...?

File: EG23.tex [hunk 4]
```diff
@@ -465,154 +440,117 @@ \subsection{Case 1: Greedy Separations}
...
+    \subfigure[Case 1.\label{fig:ARR_2}] {\epsfig{figure=img/AnnualReprocessingRate_2,width=0.48\textwidth}}
```
Comment (gonuke) [inline EG23.tex]: "Case 1" label is used for both sub-plots

**Comment by gonuke** [comment]
Some problems with figures... I need that fixed before I can review properly

**Comment by bam241** [comment]
@gonuke sorry about the figure, I was sure I had this good...
I fix it this morning...

**Pull request merged**